examples/angular_bazel_architect/e2e/protractor.conf.js

```diff
@@ -10,13 +10,21 @@ const {SpecReporter} = require('jasmine-spec-reporter');
 exports.config = {
   allScriptsTimeout: 11000,
   specs: ['./src/**/*.e2e-spec.ts'],
-  capabilities: {'browserName': 'chrome'},
   directConnect: true,
   baseUrl: 'http://localhost:4200/',
   framework: 'jasmine',
   jasmineNodeOpts: {showColors: true, defaultTimeoutInterval: 30000, print: function() {}},
   onPrepare() {
     require('ts-node').register({project: require('path').join(__dirname, './tsconfig.json')});
     jasmine.getEnv().addReporter(new SpecReporter({spec: {displayStacktrace: true}}));
+  },
+  capabilities: {
+    browserName: 'chrome',
+    pageLoadStrategy: 'normal',
+    chromeOptions: {
+      args: [
+        '--headless', '--disable-gpu', '--window-size=800,600', '--debuggerAddress=127.0.0.1:12633'
```

Comment:
Here's what I used in an angular PR that I was working on that covers most of the issues that can come up with headless and on different CI platforms:
```
args: ['--no-sandbox', '--headless', '--disable-gpu', '--disable-dev-shm-usage', '--hide-scrollbars', '--mute-audio']
```
Why the `--windows-size` and `--debuggerAddress` here? Needed for this to work?

Reply:
Documented where those args came from here: https://github.com/angular/angular/pull/33927/files#diff-303c8e2af10b19172edb0209e673a17b if you're curious

Reply:
Discussed offline & cleaned up.